server/services/email-campaigns/inactive-workplaces/findInactiveWorkplaces.js

```diff
@@ -0,0 +1,33 @@
+const nextEmail = require('./nextEmail');
+const {
+  getInactiveWorkplaces,
+  refreshInactiveWorkplaces,
+} = require('../../../models/email-campaigns/inactive-workplaces/getInactiveWorkplaces');
+
+const transformInactiveWorkplaces = (inactiveWorkplace) => {
+  const emailTemplate = nextEmail.getTemplate(inactiveWorkplace);
+
+  return {
+    id: inactiveWorkplace.EstablishmentID,
+    name: inactiveWorkplace.NameValue,
+    nmdsId: inactiveWorkplace.NmdsID,
+    lastUpdated: inactiveWorkplace.LastUpdated,
+    emailTemplate: emailTemplate,
```

Comment:
You can just use it once in ES6+ 
```suggestion
    emailTemplate,
```